src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -34,6 +34,11 @@ class FlattenException
     private $line;
 
     public static function create(\Exception $exception, $statusCode = null, array $headers = array())
+    {
+        return static::createFromThrowable($exception, $statusCode, $headers);
+    }
+
+    public static function createFromThrowable(\Throwable $exception, ?int $statusCode = null, array $headers = array()): self
```

Comment:
What about using `new` instead (`FlattenException::new()` looks better to me)?

Reply:
its not possible to call a function "new" because the keyword is reserved...

Reply:
Works in php 7 https://3v4l.org/N5iEr

Reply:
Well, then the developer would have the choice between `create()` and `new()` which looks rather confusing to me. Also, I would not recommend to use keywords as method names even if it currently works.